app/templates/classquizzes.html

```diff
@@ -0,0 +1,31 @@
+﻿{% block style %}
+<link rel="stylesheet" href="{{ url_for('static', filename='content/quizstyle.css') }}">
+{% endblock %}
+{% block content %}
```

Comment:
So in this case the quiz will not appear with the sidebar from the class right? Will be similar to the mockup/preview of our own quiz?

Reply:
Shouldn't.